### PR TITLE
BCDA-1417: Fix BB client logging statements for responses

### PR DESCRIPTION
**Pull request by embh**
### Fixes [BCDA-1417](https://jira.cms.gov/browse/BCDA-1417)
Log entries for Blue Button responses are missing values. 

### Proposed changes:
Add missing values and remove fields where the value is unknown.

### Change Details
* Uses headers from the request to populate `bb_query_id`, `bb_query_ts`, `bb_uri`
* Removes `content_length` because Go will always set it to -1
* Adds `jobID` parameter to `getData` calls from `GetPatientData` and `GetCoverageData`
* Shortens messages to simply `request` or `response` (previously 'Blue Button request' and 'Blue Button response`) as these entries are part of the Blue Button request log and have other indicators that they are to/from BB
* Separates request and response logging into two functions, called before the request is made and after the response is received

### Security Implications
No new information has been added; all values being logged in responses are already logged under requests. The Blue Button URI may contain a Blue Button patient ID or hashed HICN.

### Acceptance Validation
Log entry examples of requests to and responses from BB dev environment. `beneficiary` ID is synthetic.
Screen Shot 2019-06-13 at 1 28 45 PM
Screen Shot 2019-06-14 at 12 19 02 PM

### Feedback Requested
Any

### CODE REVIEW

**Comment by embh** [comment]
Content length is removed here: https://github.com/CMSgov/bcda-app/pull/294/files#diff-ed36edc0ce96fd7e78c1bc475cca12c5L199

File: bcda/client/bluebutton.go [hunk 1]
```diff
@@ -187,17 +187,16 @@ func logRequest(req *http.Request, resp *http.Response, jobID string) {
...
-	}).Infoln("Blue Button request")
+	}).Infoln("request")
 
 	if resp != nil {
 		logger.WithFields(logrus.Fields{
 			"resp_code":      resp.StatusCode,
-			"bb_query_id":    resp.Header.Get("BlueButton-OriginalQueryId"),
-			"bb_query_ts":    resp.Header.Get("BlueButton-OriginalQueryTimestamp"),
-			"bb_uri":         resp.Header.Get("BlueButton-OriginalUrl"),
+			"bb_query_id":    req.Header.Get("BlueButton-OriginalQueryId"),
```
Comment (tbellj) [inline bcda/client/bluebutton.go]: shouldn't these values be set from the resp param and not the req??

Comment (embh) [reply bcda/client/bluebutton.go]: Those headers used to get populated in the BB responses I think (which is why we had it set up that way originally) but don't anymore. So request is the only place where we can get them.

Comment (tbellj) [reply bcda/client/bluebutton.go]: oh okay, so BB changed the way they pass the data back to us. Gotcha

**Comment by tbellj** [comment]
one more minor tiny change, but could the screenshot reflect both a call to getPatientData and getCoverageData??

**Pull request merged**